docs/source/figures.rst

```diff
@@ -0,0 +1,14 @@
+***********
+Simulations
+***********
+
+.. currentmodule:: seirmo
+
+The models provide a framework for the implementation of any models.
```

Comment:
This doesn't really say what the plots are about does it? 😃 
I would maybe write something like: "The figures module in seirmo includes a number of visualisation classes that are popular in epidemiological modelling." (Or ant variant of that)